subgraph.yaml

```diff
@@ -5,16 +5,17 @@ schema:
   file: ./schema.graphql
 dataSources:
   - kind: cosmos
-    name: CosmosHub
+    name: Osmosis
     network: osmosis-1
     source:
-      startBlock: 0
+      startBlock: 2522560
```

Comment:
I let the example run for some time and there were no swaps. Probably that functionality was added later on, so to avoid any waiting for the users running it I just put a higher height.

Reply:
Cool, let's just leave it then.